### ISSUE TITLE
Can you share the Hyperparameters of evaluated attacks in the leaderborad?

### ISSUE THREAD

**Issue by vtddggg** [yaodongyu/TRADES#18]
Where can I get the Hyperparameters (such as steps, n_restarts) of evaluated methods in [leaderboard](https://github.com/yaodongyu/TRADES#white-box-leaderboard-1) ?.

I test Deepfool to attack TRADES and got acc of 54%, but in your leaderboard,  Deepfool linf only got 61.38%. It seems strange. Maybe some of hyperparameters are different. I want to use some of your result in my paper if you can share more informantion.

**Comment by yaodongyu** [comment]
Thanks for your interest in our work. 

For evaluating the Deepfool attack, we directly applied the default hyperparameters in Foolbox (https://github.com/bethgelab/foolbox). It is possible that you could achieve better attack results by tuning the parameters (as long as satisfies the perturbation constraint).

Let me know if you have any other questions.
Yaodong

**Comment by vtddggg** [comment]
> Thanks for your interest in our work.
> 
> For evaluating the Deepfool attack, we directly applied the default hyperparameters in Foolbox (https://github.com/bethgelab/foolbox). It is possible that you could achieve better attack results by tuning the parameters (as long as satisfies the perturbation constraint).
> 
> Let me know if you have any other questions.
> Yaodong

Thanks for your response.
Sorry that there's one more question for me:
The top1 attack ODI-PGD gets 53.01% robust accuracy, but I cant reproduce the result. If you know how many steps and num of restarts here? I try 100 steps and 20 restarts but only get 54.5%

**Comment by hongyanz** [comment]
Hi vtddggg,

The ODI-PGD adversarial example file on our GitHub leaderboard is submitted by Yusuke Tashiro and we do not know exactly the hyper-parameter details on how to generate these adversarial examples. Perhaps a best way is to ask the authors of ODI-PGD for your concern.
